universal-application-tool-0.0.1/app/views/style/BaseStyles.java

```diff
@@ -60,7 +59,8 @@
           Styles.TEXT_LG);
 
   /** For use on `input` elements that are not of type "checkbox" or "radio". */
```

Comment:
General note, I see this is specifically for checkbox and radio questions.  Have we verified whether any other questions also have this issue for large inputs?  In particular, maybe the other multi-option question type dropdown